src/main/java/service/strategy/strategyimpl/SupplyOperation.java

```diff
@@ -0,0 +1,13 @@
+package service.strategy.strategyimpl;
+
+import dao.Storage;
+import model.FruitRecord;
+
+public class SupplyOperation implements OperationHandler{
```

Comment:
The class `SupplyOperation` should be in a package named `strategy` to follow the project structure conventions outlined in the checklist. Please move this class to the correct package.